### PR TITLE
feat: add DM Sans Regular

### LINKED ISSUE #747: Include DM Sans regular font weight
DM Sans regular font weight needs to be included in the design system css

### PR DESCRIPTION
**Pull request by houbly**
- Converted typography page to markdown
- Updated DM Sans copy
- Included DM Sans regular font
- Included missing webfonts for noto sans display italic & bold italic
- Amended affected font styles in header + footer

fixes #747
fixes #748

### CODE REVIEW

**Review by gldgrnt — DISMISSED** [review]
👍🏾

File: src/www/pages/styles/typography/index.njk.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+- 55.2% of respondents preferred increased line spacing, with a further 12.1% having no preference
+
+As a public sector organisation, in addition to testing the above we are required to meet []<a href="https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag#applying-wcag-21-guidelines" target="_blank">WCAG 2.1 guidelines at AA standard</a>.
+Therefore, features will be useable when text size is increased up to 200% and content reflowed for 400%.
+
+{# Fonts #}
+
+## Fonts
+
+We utilise <a href="https://fonts.google.com/" target="_blank">Google Fonts</a> in our typography and all fonts used are pre-bundled in our design system CSS, so you won't need to include them seperately.
```
Comment (gldgrnt) [inline src/www/pages/styles/typography/index.njk.md:26]: Just this typo:

```suggestion
We utilise <a href="https://fonts.google.com/" target="_blank">Google Fonts</a> in our typography and all fonts used are pre-bundled in our design system CSS, so you won't need to include them separately.
```

**Review by gldgrnt — CHANGES_REQUESTED** [review]
All looks good except for one typo!

**Review by gldgrnt — APPROVED** [review]
👍🏾

**Review by catiarodriguescosta — APPROVED** [review]
👍

**Pull request merged**